src/main/java/org/cactoos/collection/Solid.java

```diff
@@ -52,11 +56,18 @@ public Solid(final T... array) {
      */
     public Solid(final Iterable<T> src) {
         super(
-            new CollectionOf<>(
+            new Unchecked<>(
                 new org.cactoos.scalar.Solid<>(
-                    () -> new Synced<>(new Sticky<>(src))
+                    new org.cactoos.scalar.Sticky<>(
```

Comment:
@fanifieiev if I'm right, this replacement is also out of scope (`CollectionOf should be deleted`). Maybe it's better to leave a todo?

Reply:
@iakunin Well, ok.